weaviate/auth.py

```diff
@@ -33,12 +33,12 @@ def __init__(self, client_secret: str):
         Parameters
         ----------
         client_secret : str
-            The access token.
+            The client secret credentials, NOT access token.
         """
 
         super().__init__()
         self._credentials_body["grant_type"] = "client_credentials"
-        self.client_secret_encoded = base64.b64encode(client_secret.encode('ascii')).decode('ascii')
+        self._client_secret_encoded = base64.b64encode(client_secret.encode('utf-8')).decode('utf-8')
```

Comment:
Can you add a test that needs utf-8 encoding?

Reply:
Sure, I will add one.

Reply:
done